src/components/Table/TableToolbar/TableToolbarSVGButton.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { settings } from '../../../constants/Settings';
+import { keyCodes } from '../../../constants/KeyCodeConstants';
+
+const { iotPrefix } = settings;
+
+const propTypes = {
+  onClick: PropTypes.func.isRequired,
+  testId: PropTypes.string.isRequired,
+  children: PropTypes.node.isRequired,
+};
+
+/**
+ * Toolbar button that accepts a Carbon react icon as children
+ */
+const TableToolbarSVGButton = ({ onClick, testId, children }) => {
+  return (
+    <div
+      className={`${iotPrefix}--tooltip-svg-wrapper`}
+      onClick={onClick}
+      role="button"
+      tabIndex={0}
+      onKeyDown={e => {
+        if (e.keyCode === keyCodes.ENTER) onClick();
+      }}
+      data-testid={testId}
+    >
+      {children}
+    </div>
```

Comment:
What's the reason for not just using an IconOnly button from Carbon? [They spread `...other`](https://github.com/carbon-design-system/carbon/blob/468d6ac9a524b031f920bcb8048b8c4e3e865823/packages/react/src/components/Button/Button.js#L33), so we should be able to put all these props `onKeyDown`, `tabIndex`, `data-testid`, etc. on it.

Reply:
I don't know what the original reason was. I simply refactored out some duplicate code into this component and I wanted the HTML output to be the same to avoid introducing new regression issues. 

We can change it, but perhaps it is better to do it in a separate issue?

Reply:
Opened #1174